myblog/forms.py

```diff
@@ -0,0 +1,7 @@
+from django import forms
+from . import models
+
+class PostForm(forms.ModelForm):
+    class Meta:
+        model = models.Post
```

Comment:
Dado a mudança a acima, aqui ficaria: 
```suggestion
        model = Post
```
O que acha?